packages/@ember/-internals/glimmer/tests/integration/outlet-test.js

```diff
@@ -9,8 +9,9 @@ moduleFor(
       super(...arguments);
 
       let CoreOutlet = this.owner.factoryFor('view:-outlet');
-
-      this.component = CoreOutlet.create();
+      let outletTemplateFactory = this.owner.lookup('template:-outlet');
+      let environment = this.owner.lookup('-environment:main');
+      this.component = CoreOutlet.create({ environment, template: outletTemplateFactory });
```

Comment:
☝️ this is important to note.  This was refactored from the implicit injection.

Reply:
I believe this  needs to be ported to `@ember/test-helpers` https://github.com/emberjs/ember-test-helpers/blob/9219247259ef6b4d8a339ca71a1374e2ffe69603/addon-test-support/%40ember/test-helpers/setup-rendering-context.ts#L266

Reply:
working on that now.

Reply:
https://github.com/emberjs/ember-test-helpers/pull/1110

Reply:
landed and released as @ember/test-helpers@2.4.2